packages/extension/src/App.tsx

```diff
@@ -74,12 +74,14 @@ const App: FC = () => {
 
       if (extensionWindow) {
         chrome.runtime
-          .sendMessage<ReceiveNetworkBackgroundMessage>({
+          .sendMessage<ReceiveGetNetworkBackgroundMessage>({
             app: GEM_WALLET,
-            type: 'RECEIVE_NETWORK',
+            type: 'RECEIVE_GET_NETWORK/V3',
```

Comment:
You need to be retro-compatible, have a look at the `Login` component to see how to handle it.

Reply:
Done